binding/flutter/README.md

```diff
@@ -0,0 +1,112 @@
+# Leopard Binding for Flutter
+
+## Leopard Speech-to-Text Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Leopard is an on-device speech-to-text engine. Leopard is:
+
+- Private; All voice processing runs locally.
+- Accurate [[1]](https://github.com/Picovoice/speech-to-text-benchmark#results)
+- Compact and Computationally-Efficient [[2]](https://github.com/Picovoice/speech-to-text-benchmark#results)
+- Cross-Platform:
+    - Linux (x86_64)
+    - macOS (x86_64, arm64)
+    - Windows (x86_64)
+    - Android
+    - iOS
+    - Raspberry Pi (4, 3)
+    - NVIDIA Jetson Nano
+
+## Compatibility
+
+This binding is for running Leopard on **Flutter 1.20.0+** on the following platforms:
+
+- Android 4.4+ (API 19+)
```

Comment:
isn't android min requiredment 5.0?

Reply:
I didn't think so

Reply:
https://github.com/Picovoice/leopard/tree/master/binding/android we have it as 5.0 as well as for cheetah i think.